cmd/query/app/server.go

```diff
@@ -347,13 +353,31 @@ func (s *Server) Start() error {
 // Close stops http, GRPC servers and closes the port listener.
 func (s *Server) Close() error {
 	var errs []error
+
+	// Log and close TLS gRPC server
```

Comment:
comment is redundant (and invalid, we're not closing the server itself, but a TLS config watcher)